documentation/administration/Portal_Administration_sv_SE.md

```
@@ -22,9 +22,9 @@ Portal-API-användaren är ett särskilt användarkonto som skapats specifikt f
 
 ### Specifika rekommenderade behörigheter
 
-#### Logiska enheter (AIP)
-- Hämta logiska enheter (AIP)
-- Lista och sök efter logiska enheter (AIP)
+#### Förvaringsenheter (AIP)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)